sessionctx/variable/session.go

```diff
@@ -26,10 +27,12 @@ const (
 	codeCantGetValidID terror.ErrCode = 1
 	codeCantSetToNull  terror.ErrCode = 2
 	codeSnapshotTooOld terror.ErrCode = 3
+	codeTruncated      terror.ErrCode = terror.ErrCode(mysql.WarnDataTruncated)
```

Comment:
You mean `ErrTruncated` in `util/types/errors.go`? `sessionctx/variable` package can't reference that variable, because that will cause `cycle import error`. Any better solution for that?